apps/fz_http/lib/fz_http/users/user.ex

```diff
@@ -155,6 +144,14 @@ defmodule FzHttp.Users.User do
     |> validate_required([:role])
   end
 
+  # Password reset token
+  def update_changeset(
+        user,
+        %{sign_in_token: _token, sign_in_token_created_at: _created_at} = attrs
+      ) do
+    cast(user, attrs, [:sign_in_token, :sign_in_token_created_at])
+  end
```

Comment:
Changed to use atoms, as they are system internals.

Aside: we don't necessarily want to support both strings and atoms in all cases. Strings and atoms are clear boundaries that something is coming from the outside world or from the system itself.

Reply:
Good call. Makes pattern matching functions more clear too. This is an area I've been wanting to refactor esp since it handles authentication.